src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.cs

```diff
@@ -65,10 +67,20 @@ public void All_event_types_are_round_trip_serializable(KernelEvent @event)
 
             var deserializedEnvelope = KernelEventEnvelope.Deserialize(json);
 
+            // ignore these specific properties because they're not serialized
+            var ignoredProperties = new HashSet<string>()
+            {
+                $"{nameof(CommandFailed)}.{nameof(CommandFailed.Exception)}",
+                $"{nameof(DisplayEvent)}.{nameof(DisplayEvent.Value)}"
```

Comment:
`DisplayEvent` is abstract so what about e.g. `ReturnValueProduced`?

Reply:
I initially tried that, but since `DisplayEvent` is the declaring type, this is the only thing that matches.

Reply:
Also, due to how that method is implemented, I have access either to the type information through `SelectedMemberInfo` like I currently do, _or_ I can use the `EventType` property, but not both, and since the member name (`SelectedMemberInfo.Name`) is the really important part, we have to fall back to this.